test/e2e/capi_test.go

```diff
@@ -241,6 +241,7 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 						InitWithBootstrapProviders:      []string{"kubeadm:v1.0.5"},
 						InitWithControlPlaneProviders:   []string{"kubeadm:v1.0.5"},
 						InitWithInfrastructureProviders: []string{"azure:v1.0.2"},
+						// Note: this probably won't work with CAAPH since the Helm install only works with CAPI/clusterctl v1.5.0.
```

Comment:
@willie-yao I saw you made some changes here, I don't think we can use the CAAPH install with CAPI v1.0.5. Is there any way we can get around this?

Reply:
Is there a way to add an exception to not use CAAPH for this test? This test is testing the upgrade from the earliest stable version of CAPI to current, so it will be useful to keep around. If not, is there a minimum version of CAPI that will work with CAAPH install?